frontend/src/layouts/SideNav.vue

```diff
@@ -1,13 +1,49 @@
 <script setup lang="ts">
+import { useQuery } from "@tanstack/vue-query";
+import { AdminRoleAuthGroup } from "fam-admin-mgmt-api/model";
 import { useRouter, type RouteRecordName } from "vue-router";
 import { sideNavState } from "@/store/SideNavState";
 import EmailIcon from "@carbon/icons-vue/es/email/16";
+import AlignBoxIcon from "@carbon/icons-vue/es/align-box--top-center/16";
 import Sidebar from "primevue/sidebar";
 import { sideNavItems } from "@/constants/SideNavConfig";
 import Label from "@/components/UI/Label.vue";
+import { AdminMgmtApiService } from "@/services/ApiServiceFactory";
+import { computed } from "vue";
 
 const router = useRouter();
 
+const adminUserAccessQuery = useQuery({
+    queryKey: ["admin-user-access"],
+    queryFn: () =>
+        AdminMgmtApiService.adminUserAccessesApi
+            .adminUserAccessPrivilege()
+            .then((res) => res.data),
+});
+
+/**
+ * Determines if a user is an app admin for at least one application,
+ * @returns {string} a path to the relevant pdf file.
+ */
+const pathToPdfGuide = computed(() => {
+    if (!adminUserAccessQuery.data.value) {
+        return "";
+    }
+
+    const accessList = adminUserAccessQuery.data.value.access.map(
+        (grantDto) => grantDto.auth_key
+    );
+
+    const famAdminIndex = accessList.indexOf(AdminRoleAuthGroup.FamAdmin);
```

Comment:
This needs to be `AdminRoleAuthGroup.AppAdmin` for the condition, like:
```
const isAppAdmin = accessList.indexOf(AdminRoleAuthGroup.AppAdmin) > -1;
```

Reply:
Hi @craigyu , hope you don't mind, I slightly adjust the conditions and also hide the link when user only is a FAM Admin.